Sources/DiscoverKit/View/RecommendationsView.swift

```diff
@@ -8,15 +8,25 @@ struct RecommendationsView: View {
     let recommendations: [Recommendation]
     @EnvironmentObject var viewModel: DiscoverViewModel
 
+    @Environment(\.horizontalSizeClass)
+    var sizeClass
+
+    static let maxReadableWidth: CGFloat = 700
```

Comment:
can we add a comment that explains why we are using this particular number?

Reply:
okay! I can also link to the PR where this number originally came from.